guide/ops/catalog/index.md

```diff
@@ -161,6 +167,30 @@ The following optional catalog metadata is supported:
   whenever a bundle is reloaded in a Brooklyn server,
   and if entities have been deployed against that version, their behavior may change in subtle or potentially incompatible ways.
   To avoid this situation, it is highly recommended to use OSGi version stamps as part of the URL.
+- To refer a resource from a JAR file added by brooklyn.libraries 
+  you should include OSGi Bundle-SymbolicName in the classpath.
+  Example:
+  
+~~~ yaml
+brooklyn.catalog:
+  brooklyn.libraries:
+  - url: http://some.server.or.other/path/my.jar
+    name: myJar-BundleSymbolicName
+    version: 1.0
+  items:
+  - classpath://myJar-BundleSymbolicName:my-catalog-entries-inside-jar.bom
+  - id: use-from-my-catalog
+    version: "1.0.0"
+    itemType: entity
+    iconUrl: classpath://myJar-BundleSymbolicName:org/apache/brooklyn/MyEntity/icon.png
+    item:
+      type: myJar-BundleSymbolicName:org.apache.brooklyn.MyEntity
+      brooklyn.config:
+        some.config: "some value"
+        templates.install:
+          classpath://myJar-BundleSymbolicName:org/apache/brooklyn/MyEntity/config.xml: config.xml 
```

Comment:
We should try to fix cases where the prefix is needed. Does the example work without the prefixes? If not can you file a bug against Brooklyn.